motoman_driver/src/industrial_robot_client/joint_trajectory_action.cpp

```diff
@@ -159,10 +160,20 @@ JointTrajectoryAction::JointTrajectoryAction() :
 
     this->act_servers_[group_number_int]->start();
 
+    this->act_clients_[group_number_int] = actionClient_;
```

Comment:
Can you explain why action clients are needed in this node?  Originally, this node contained just servers that sent commands (via topics) to the robot driver node.  What do the client connect to?